addon/components/drag-sort-item.js

```diff
@@ -76,6 +77,12 @@ export default Component.extend({
     return draggingEnabled ? true : null
   }),
 
+
+  listHasDisabledSorting : computed('disableSorting', (disableSorting) => {
+    return disableSorting ? true : null
+  }),
```

Comment:
What's the point of this CP? Why not use `disableSorting` directly?

It may be useful to add a conditional HTML class, but I don't see you do that.

Also, this addon uses `ember-awesome-macros`, so this CP could be rewritten as:

```
import conditional from 'ember-awesome-macros/conditional'

{
  listHasDisabledSorting : conditional('disableSorting', true, null)
}
```